### PR TITLE
Add a GSVA wrapper script and the associated galaxy tool config file.

### PR DESCRIPTION
**Pull request by jdh33**

### CODE REVIEW

**Comment by alliecreason** [comment]
Suggest integrating the R wrapper into the galaxy xml tool: https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-configfiles

**Comment by jdh33** [comment]
Move the R script code into the xml config file as Cheetah code. 
Add image for help section.
Add test data.

File: gsva_tools/run_gsva.xml [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+<!-- TODO (hocumj 7/15/2019): Provide default gene sets. -->
+<!-- TODO (hocumj 7/15/2019): Allow combining gene sets from multiple files. -->
+<!-- TODO (hocumj 7/15/2019): Add preprocessing step to handle missing Ensembl ID versions. E.g., ENSG00000171223.5 compared to ENSG00000171223 -->
+<!-- TODO (hocumj 7/15/2019): Implement some tests for running with non-default parameters. -->
+
+<tool id="run_gsva" name="Run GSVA:" version="0.1.0">
+    <description>Estimate gene set enrichment statistics from gene expression values.</description>
+    <requirements>
+        <requirement type="package" version="3.0.0">R</requirement>
+        <requirement type="package" version="1.32.0">GSVA</requirement>
```
Comment (alliecreason) [inline gsva_tools/run_gsva.xml]: Check conda packages exist for these requirements.

File: gsva_tools/run_gsva.xml [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+<!-- Description: A simple galaxy wrapper tool to run GSVA. -->
+<!-- TODO (hocumj 7/15/2019): Consider moving some code to a macro.xml file. -->
+<!-- TODO (hocumj 7/15/2019): Handle different delimiters. -->
+<!-- TODO (hocumj 7/15/2019): Implement try/except error handling. -->
+<!-- TODO (hocumj 7/15/2019): Provide default gene sets. -->
+<!-- TODO (hocumj 7/15/2019): Allow combining gene sets from multiple files. -->
+<!-- TODO (hocumj 7/15/2019): Add preprocessing step to handle missing Ensembl ID versions. E.g., ENSG00000171223.5 compared to ENSG00000171223 -->
+<!-- TODO (hocumj 7/15/2019): Implement some tests for running with non-default parameters. -->
+
+<tool id="run_gsva" name="Run GSVA:" version="0.1.0">
```
Comment (alliecreason) [inline gsva_tools/run_gsva.xml]: Suggest changing `Run GSVA:` to either `GSVA` or `Gene Set Enrichment Analysis`. Current name can look awkward when referring to the tool in places other than the tool panel (such as the admin panel).

**Comment by alliecreason** [comment]
Please add `.shed.yml` file for uploading tool to local tool shed. See our [wiki](https://github.com/ohsu-comp-bio/compbio-galaxy-wrappers/wiki/Managing-Custom-Tools#local-toolshed-) or [planemo docs](https://planemo.readthedocs.io/en/latest/standards/docs/best_practices/shed_yml.html).

**Comment by jdh33** [comment]
All required packages could be installed using the galaxy dependency resolver.
r-base 3.6.1, r-reshape2 1.4.3, and r-dplyr 0.8.3 are installed from conda-forge.
bioconductor-gseabase 1.46.0 and bioconductor-gsva 1.32.0 and installed from bioconda.

**Pull request merged**